### PR TITLE
40071

### PR DESCRIPTION
**Pull request by saimekala07**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [40071](https://devops.dchbx.org/redmine/issues/40071) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | 2 |
| Peer Review Story Points | n/a |
| Ticket Author | @TreyE |

### CODE REVIEW

File: spec/models/carrier_spec.rb [hunk 1]
```diff
@@ -14,7 +14,8 @@
     :policies,
     :premium_payments,
     :brokers,
-    :carrier_profiles
+    :carrier_profiles,
+    :requires_reinstate_for_earlier_termination
   ].each do |attribute|
     it { should respond_to attribute }
   end
```
Comment (TreyE) [inline spec/models/carrier_spec.rb]: I don't see a spec here testing the **behaviour** of the new attribute.  What is it's default expected to be, for example?

File: script/update_requires_reinstate_for_earlier_termination_carrier.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ Carrier.all.each do |carrier|
+   unless carrier.name == "Care First"
```
Comment (TreyE) [inline script/update_requires_reinstate_for_earlier_termination_carrier.rb]: I'm not sure this is sufficient selection criteria based on what the name may be - can you use hbx ID instead?

File: spec/models/carrier_spec.rb (hunk 1 above)
Comment (saimekala07) [reply spec/models/carrier_spec.rb]: Added spec

**Comment by harshared** [comment]
already updated in branch enddate-reinstate-glue-rc, so closing this

**Pull request closed without merging**